<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SummonerProfile</class>
 <widget class="QMainWindow" name="SummonerProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="enter_summoner_label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>370</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Summoner</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="summoner_input">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>400</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="platform_input">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>400</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="choose_platform_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>370</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Platform</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submit_button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>480</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="summoner_display">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Summoner Placeholder</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="rankImage">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>210</y>
      <width>45</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="solo_group">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>271</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="solo_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>231</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="solo_placeholder">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Solo Queue Rank</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="flex_group">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>291</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="flex_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="flex_placeholder">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Flex Queue Rank</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
